src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -1,53 +1,194 @@
+/*
+ * File: z_boss_va.c
+ * Overlay: ovl_Boss_Va
+ * Description: Barinade
+ */
+
 #include "z_boss_va.h"
+#include "objects/object_bv/object_bv.h"
+#include "overlays/actors/ovl_En_Boom/z_en_boom.h"
 #include "objects/gameplay_keep/gameplay_keep.h"
 
 #define FLAGS 0x00000035
 
 #define THIS ((BossVa*)thisx)
+#define BODY ((BossVa*)this->actor.parent)
```

Comment:
Personally not a fan of this macro since it just hides important details imo, but I think it should at least be defined with the pointer as an argument:
```suggestion
#define GET_BODY(bossVa) ((BossVa*)(bossVa)->actor.parent)
```